lk2nd/device/dts/msm8953/apq8053-lenovo-cd-18781y.dts

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_APQ8053 0>;
+	qcom,board-id= <0x1010520 0>;
+};
+
+&lk2nd {
+	model = "Lenovo ThinkSmart View";
+	compatible = "lenovo,cd-18781y";
+
+    lk2nd,dtb-files = "apq8053-lenovo-cd-18781y";
```

Comment:
This is spaces not tabs, I'd appreciate if everything is consistent and uses tabs.